src/main/java/org/lwes/emitter/UDPEmitterGroupFactory.java

```diff
@@ -0,0 +1,210 @@
+package org.lwes.emitter;
```

Comment:
Is this pretty much a rename of EmitterGroupBuilder as far as the contents are concerned?

Reply:
yes